test/components/world-map.spec.js

```diff
@@ -0,0 +1,14 @@
+import {expect} from 'chai'
+import {render} from 'enzyme'
+import React from 'react'
+
+import WorldMap from '../../app/scripts/components/world-map'
+
+describe('WorldMap', () => {
+  it('should render an svg', () => {
+    const c = [[12, 14]]
+
+    const el = render(<WorldMap coordinates={c}/>)
+    expect(el.find('svg').length).to.equal(1)
```

Comment:
I tried to think of more meaningful assertions to add here but drew a blank. The resulting output of `_renderMap` is basically all generated by `d3` and I couldn't find anything else on the rendered output that could be used to assert correctness.